source/_docs/z-wave/entities.markdown

```diff
@@ -0,0 +1,179 @@
+  - **6**: Access control
+
+### {% linkable_title Alarm Level %}
+
+The meaning of the `alarm_level` entity depends on the nature of the alarm sensor
+
+#### {% linkable_title Smoke, CO, and CO2 %}
+
+  - **1**: Detection - will include a Node Location Report
+  - **2**: Detection (unknown location)
+  - **254**: Unknown event
+
+#### {% linkable_title Heat %}
+
+  - **1**: Overheat detected - will include a Node Location Report
+  - **2**: Overheat detected (unknown location)
+  - **3**: Rapid temperature rise - will include a Node Location Report
+  - **4**: Rapid temperature rise (unknown location)
+  - **5**: Underheat detection - will include a Node Location Report
+  - **6**: Underheat detection (unknown location)
+  - **254**: Unknown event
+
+#### {% linkable_title Water leak %}
+
+  - **1**: Water leak detected - will include a Node Location Report
+  - **2**: Water leak detected (unknown location)
+  - **3**: Water level dropped - will include a Node Location Report
+  - **4**: Water level dropped (unknown location)
+  - **254**: Unknown event
+
+#### {% linkable_title Access control %}
+
+  - **1**: Manual lock
+  - **2**: Manual unlock
+  - **3**: RF lock
+  - **4**: RF unlock
+  - **5**: Keypad lock - will include the User Identifier of the User Code Report
+  - **6**: Keypad unlock - will include the User Identifier of the User Code Report
+  - **254**: Unknown event
+
+#### {% linkable_title Burglar %}
+
+  - **1**: Intrusion - will include a Node Location Report
+  - **2**: Intrusion (unknown location)
+  - **3**: Tampering (case opened)
+  - **4**: Tampering (invalid code)
+  - **5**: Glass break - will include a Node Location Report
+  - **6**: Glass break (invalid code)
+  - **254**: Unknown event
+
+#### {% linkable_title Power Management %}
+
+  - **1**: Power applied
+  - **2**: AC disconnected
+  - **3**: AC re-connected
+  - **4**: Surge detection
+  - **5**: Voltage drop or drift
+  - **254**: Unknown event
+
+#### {% linkable_title System Alarm %}
+
+  - **1**: System hardware failure
+  - **2**: System software failure
+  - **254**: Unknown event
+
+#### {% linkable_title Emergency Alarm %}
+
+  - **1**: Contact Police
+  - **2**: Contact Fire Service
+  - **3**: Contact Medical Service
+  - **254**: Unknown event
+
+#### {% linkable_title Alarm Clock %}
+
+  - **1**: Wake up
+  - **254**: Unknown event
+
+### {% linkable_title Access Control %}
+
+- **access_control**: These *may* vary between brands
+	 - **22**: Open
+	 - **23**: Closed
+	 - **254**: Deep sleep
+	 - **255**: Case open
+
+If your device has an `access_control` entity, but not a `binary_sensor` equivalent, you can use a [template binary sensor](omponents/binary_sensor.template/) to create one:
+
+```
+sensor:
+  - platform: template
+    sensors: 
+      YOUR_SENSOR:
+        sensor_class: opening
+        friendly_name: "Friendly name here"
+        value_template: >- 
+          {%- if is_state('sensor.YOUR_SENSOR_access_control', '22') -%}
```

Comment:
I think access_control should have the if state written to 23 - closed.  This was if it's not 23, it's not closed.

Otherwise, either 254 or 255 would show as closed